{{cookiecutter.python_name}}/pyproject.toml

```
@@ -1,18 +1,12 @@
 [build-system]
-requires = ["hatchling>=1.3.1", "jupyterlab~=3.1"]
+requires = ["hatchling>=1.3.1", "jupyterlab~=3.1", "hatch-nodejs-version"]
```

Ensure jupyterlab 3.4.7 or later is used

```suggestion
requires = ["hatchling>=1.3.1", "jupyterlab>=3.4.7,<4.0.0", "hatch-nodejs-version"]
```